chapter2/tests/t12/nginx.conf

```diff
@@ -0,0 +1,74 @@
+user nginx nginx;
+worker_processes 1;
+error_log /tmp/camp/error.log notice;
+error_log /tmp/camp/error_debug.log debug;  
+pid /run/nginx.pid;
+
+events {
+	worker_connections 1024;
+}
+
+http {
+
+	log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+					  '$status $body_bytes_sent "$http_referer" '
+					  '"$http_user_agent" "$http_x_forwarded_for"';
```

Comment:
find a way to format this better.

Reply:
> find a way to format this better.

I partially corrected it, but not the way I wanted to. The display behaves very strangely in Github, while in VSCode everything looks normal. I haven't figured out the cause yet, but from now on I'll double-check with my eyes after every push..
![Screenshot from 2023-04-27 23-25-46](https://user-images.githubusercontent.com/93929010/234985024-72e0c682-9041-4127-9de0-f2804f9b6420.png)